plugins/plugin-machine/che-plugin-machine-ext-server/src/test/java/org/eclipse/che/ide/ext/machine/server/ssh/KeysInjectorTest.java

```diff
@@ -10,284 +10,265 @@
-  // echo 'publicKey1' >> ~/.ssh/authorized_keys"});
-  //        verify(docker).startExec(eq(StartExecParams.create(EXEC_ID)), anyObject());
-  //        verifyZeroInteractions(docker, environmentEngine, sshManager);
-  //    }
-  //
-  //    /**
-  //     * Validate the usecase of no workspace keypair (notfound exception) and no machine keypair
-  //     * Expect no ssh keys are injected
-  //     */
-  //    @Test
-  //    public void shouldNotInjectSshKeysWhenThereIsNoWorkspaceKey() throws Exception {
-  //        // no machine key pairs
-  //        when(sshManager.getPairs(anyString(), eq("machine")))
-  //                .thenReturn(Collections.emptyList());
-  //
-  //        // no workspace keypair
-  //        when(sshManager.getPair(anyString(), eq("workspace"), anyString()))
-  //                .thenThrow(NotFoundException.class);
-  //
-  //
-  //
-  // subscriber.onEvent(newDto(MachineStatusEvent.class).withEventType(MachineStatusEvent.EventType.RUNNING)
-  //                                                           .withMachineId(MACHINE_ID)
-  //                                                           .withWorkspaceId(WORKSPACE_ID));
-  //
-  //        verify(environmentEngine).getMachine(eq(WORKSPACE_ID), eq(MACHINE_ID));
-  //        // check calls for machine and workspace ssh pairs
-  //        verify(sshManager).getPairs(eq(OWNER), eq("machine"));
-  //        verify(sshManager).getPair(eq(OWNER), eq("workspace"), eq(WORKSPACE_ID));
-  //
-  //        verifyZeroInteractions(docker, environmentEngine, sshManager);
-  //    }
-  //
-  //    @Test
-  //    public void shouldSendMessageInMachineLoggerWhenSomeErrorOcursOnKeysInjection() throws
-  // Exception {
-  //        when(sshManager.getPairs(anyString(), anyString()))
-  //                .thenReturn(Collections.singletonList(new SshPairImpl(OWNER, "machine",
-  // "myPair", "publicKey1", null)));
-  //        when(logMessage.getType()).thenReturn(LogMessage.Type.STDERR);
-  //        when(logMessage.getContent()).thenReturn("FAILED");
-  //
-  //
-  // subscriber.onEvent(newDto(MachineStatusEvent.class).withEventType(MachineStatusEvent.EventType.RUNNING)
-  //                                                           .withMachineId(MACHINE_ID)
-  //                                                           .withWorkspaceId(WORKSPACE_ID));
-  //
-  //        verify(docker).startExec(eq(StartExecParams.create(EXEC_ID)),
-  // messageProcessorCaptor.capture());
-  //        final MessageProcessor<LogMessage> value = messageProcessorCaptor.getValue();
-  //        value.process(logMessage);
-  //
-  //        verify(lineConsumer).writeLine(eq("Error of injection public ssh keys. FAILED"));
-  //    }
+
+  private static final String WORKSPACE_ID = "workspace123";
+  private static final String OWNER = "user123";
+
+  @Captor private ArgumentCaptor<EventSubscriber<WorkspaceStatusEvent>> subscriberCaptor;
+
+  @Mock Exec exec;
```

Comment:
As far as I see `exec` and `logMessage` fields are not used anywhere. Please review tests.